client/src/components/pipelines/browser/DataStorage.js

```diff
@@ -35,6 +35,7 @@ import {
   Spin,
   Table
 } from 'antd';
+import dataStorageCache from '../../../models/dataStorage/DataStorageCache';
```

Comment:
This import is redundant - `dataStorageCache` should be injected (it is initialized in `Root.js`)